src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -5327,4 +5327,7 @@ nuget locals global-packages -list</value>
     <value>nuget restore MySoluion.sln</value>
     <comment>command line example, do not localize</comment>
   </data>
+  <data name="CommandDirectDownload" xml:space="preserve">
+    <value>Download directly without populating the global package cache</value>
```

Comment:
This piece of text is not true for `V3` restores. V3 restores always populate the global package cache.

nit: should have a trailing period to match `NoCache` text.